VolleyPaint/Assets/Scripts/ProjectileLaunch.cs

```diff
@@ -0,0 +1,136 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Blobcreate.Universal;
+using Blobcreate.ProjectileToolkit;
+
+[CreateAssetMenu(menuName = "Litkey/Ability/Projectile/ManageBall")]
+public class ProjectileLaunch : Ability
```

Comment:
could probably rename it to something like SlingShot if you want to

Reply:
Makes sense to me. Got it.